sdk/script/management-sdk.ts

```diff
@@ -52,13 +55,15 @@ class AccountManager {
     private _accessKey: string;
     private _serverUrl: string;
     private _customHeaders: Headers;
+    private _proxy: string;
 
-    constructor(accessKey: string, customHeaders?: Headers, serverUrl?: string) {
+    constructor(accessKey: string, customHeaders?: Headers, serverUrl?: string, proxy?: string) {
```

Comment:
I'd be in favor of converting the last three parameters into an options object since they're all optional. That way, this constructor only has two parameters, and a user wouldn't need to specify two nulls simply to specific a proxy server.